### PR TITLE
Improve the layout of LogChimp's documentation pages

### PR DESCRIPTION
**Pull request by okezieuc**
* Makes the sidebar of docs page fixed
* Makes the header fixed on desktop and tablets

Link to demo: https://website-git-improve-layout-okezieuc.vercel.app/

### CODE REVIEW

**Comment by okezieuc** [comment]
Any changes needed or comments on the PR?

**Comment by mittalyashu** [comment]
Apologies for delay in response, looking into it now.

**Comment by mittalyashu** [comment]
The header is not 100% to the viewport.

CleanShot 2021-09-04 at 09 10 26@2x

How about we use 100% full for the docs page? Like this https://thermal.codecarrot.net/docs/introduction

**Comment by okezieuc** [comment]
The header spans the full width of the viewport on small to large screens but doesn't occupy the whole width on very large screens. This behaviour is the same as the behavior on all other pages and the live site.

**Comment by mittalyashu** [comment]
Can you try changing the layout for the `Doc` template to use full width?

**Comment by okezieuc** [comment]
I have added a CSS style that makes the docs page cover the full width of the screen. You can preview it on the same link: https://website-git-improve-layout-okezieuc.vercel.app/docs

File: src/styles/templates/docs.sass [hunk 1]
```diff
@@ -3,12 +3,45 @@
 	grid-gap: 2vmin
 	grid-template-columns: calc(200px + 4vmin) 1fr calc(220px + 2vmin)
```
Comment (mittalyashu) [inline src/styles/templates/docs.sass:4]:
```suggestion
	grid-template-columns: calc(200px + 4vmin) 800px calc(220px + 2vmin)
	justify-content: space-between
```

**Comment by mittalyashu** [comment]
@okezieuc Can you make header to full-width as well?

File: src/styles/templates/docs.sass [hunk 2]
```diff
@@ -3,12 +3,45 @@
...
+.docs-header
+	margin-left: -4vmin
+	margin-right: -4vmin
+	position: sticky
+	top: 0
+	background: white
+
+.page-divider
+	border-top: 1px solid #e5e5e5
+	margin-top: -1.5rem
```
Comment (mittalyashu) [inline src/styles/templates/docs.sass:19]:
```suggestion
	margin-top: -1.5rem
```

Why have you added this line?

Comment (okezieuc) [reply src/styles/templates/docs.sass:19]: The Header component has a style margin-bottom: 1.5rem which causes the line to fall too low. The style keeps things aligned.

without style
vs
with style

Comment (mittalyashu) [reply src/styles/templates/docs.sass:20]: Hmm... that's strange 🤔

**Comment by mittalyashu** [comment]
Can you make header to full-width as well?

**Comment by okezieuc** [comment]
I don't understand what you mean by set header to full-width. Maybe you can send in a commit for that.

**Comment by mittalyashu** [comment]
Yup, that looks gud.

Any final touch-ups? @okezieuc

File: src/styles/templates/docs.sass (hunk 2 above)
Comment (mittalyashu) [reply src/styles/templates/docs.sass:20]: Found it. This is the source https://github.com/logchimp/website/blob/master/src/styles/header.sass#L3

**Comment by okezieuc** [comment]
Looks good to me👍

**Comment by mittalyashu** [comment]
Great.

**Pull request merged**